packages/kubectl/packaging

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+VERSION=v1.15.0
+
+echo Retrieving kubetcl ${VERSION}
+
+curl -LO https://storage.googleapis.com/kubernetes-release/release/${VERSION}/bin/linux/amd64/kubectl
```

Comment:
Ideally we'd put the binary in the blobstore, and not download things as part of packaging.

Reply:
I went with what we already had somewhere else. Pretty much a copy.
We can change this in a separate ticket and PR, in all the places.
Fixed the typo I see above, with commit f3df90d8c255a2be1656283b07fa8257a1bd2080